src/modules/pos/validator/components/ClaimRewardsForm/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { useRewardsClaimableWithTokenMeta } from '@pos/reward/hooks/queries';
+import classNames from 'classnames';
+import BoxHeader from '@theme/box/header';
+import { useTranslation } from 'react-i18next';
+import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
+import TxComposer from '@transaction/components/TxComposer';
+import { useCurrentAccount } from '@account/hooks';
+import { QueryTable } from '@theme/QueryTable';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import styles from './ClaimRewardsForm.css';
+
+const rewardsClaimableHeader = (t) => [
+  {
+    title: t && t('Token'),
+    classList: `${grid['col-xs-3']}`,
+  },
+  {
+    title: t && t('Reward amount'),
+    classList: `${grid['col-xs-4']}`,
+  },
+  {
+    title: t && t('Fiat'),
+    classList: `${grid['col-xs-1']}`,
+  },
+];
+
+const RewardsClaimableRow = ({ data }) => {
+  const { tokenName, logo, reward, symbol } = data;
+  const amountInFiat = reward;
+
+  return (
+    <div className={classNames(styles.rewardsClaimableRow)}>
+      <div className={classNames(styles.logoContainer, rewardsClaimableHeader()[0].classList)}>
+        <img className={styles.logo} src={logo?.png} />
+        <span className={styles.tokenName}>{tokenName}</span>
+      </div>
+      <div className={classNames(rewardsClaimableHeader()[1].classList)}>
+        {`${reward} ${symbol}`}
```

Comment:
```suggestion
        {`${reward} ${symbol}`}
```
this could be trickly, we need to convert reward from specific decimal , in this case beddows to lsk

1. use tokens meta hook response http://165.227.246.146:9901/api/v3/blockchain/apps/meta/tokens
2. match the token and use `decimals` to convert